pytorch_lightning/utilities/seed.py

```diff
@@ -41,16 +41,14 @@ def seed_everything(seed: Optional[int] = None) -> int:
 
     try:
         if seed is None:
-            seed = os.environ.get("PL_GLOBAL_SEED", _select_seed_randomly(min_seed_value, max_seed_value))
+            seed = os.environ.get("PL_GLOBAL_SEED")
         seed = int(seed)
     except (TypeError, ValueError):
         seed = _select_seed_randomly(min_seed_value, max_seed_value)
+        rank_zero_warn(f"No correct seed found, seed set to {seed}")
 
-    if (seed > max_seed_value) or (seed < min_seed_value):
-        log.warning(
-            f"{seed} is not in bounds, \
-            numpy accepts from {min_seed_value} to {max_seed_value}"
-        )
+    if not (min_seed_value <= seed <= max_seed_value):
+        rank_zero_warn(f"{seed} is not in bounds, numpy accepts from {min_seed_value} to {max_seed_value}")
         seed = _select_seed_randomly(min_seed_value, max_seed_value)
 
     os.environ["PL_GLOBAL_SEED"] = str(seed)
```

Comment:
Can you log PL_GLOBAL_SEED in each process, so the user can assert it is properly set.

Reply:
great idea, let's do it?
@SeanNaren this PR is basically done yes? or do you need help with anything?
happy to test it when it's ready

Reply:
Back to this, yes let me add that!